src/pages/conference/timeline/_schedule.tsx

```diff
@@ -0,0 +1,176 @@
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+
+    h2:last-child {
+        margin-bottom: 0;
+    }
+`
+
+const ScheduleConference = ({ item }: ConferenceProps) => {
+    const { id, date, data, period, title } = item
+
+    return (
+        <>
+            <HeaderTitle key={id}>
+                <Header as="div" weight="400" type="subtitle-1" align="center" pt="16px">
+                    {title}
+                </Header>
+                <Header as="div" weight="400" type="subtitle-1" align="center" pt="16px">
+                    {date}
+                </Header>
+                <Header as="div" weight="400" type="subtitle-1" align="center" pt="16px">
+                    {period}
+                </Header>
+            </HeaderTitle>
+
+            <Container key={id}>
+                {data.map((item, index) => {
+                    return (
+                        <>
+                            <MainContainer>
+                                <WrapperContainer>
+                                    <TimeWrapper>
+                                        {item.timings.map((item) => {
+                                            return (
+                                                <Header
+                                                    as="div"
+                                                    type="heading-3"
+                                                    align="center"
+                                                    weight="700"
+                                                    mb="60px"
+                                                    key={id}
+                                                >
+                                                    {item.time}
+                                                </Header>
+                                            )
+                                        })}
+                                    </TimeWrapper>
+                                    <ImageContainer>
+                                        {index % 2 === 0 ? (
+                                            <img src={ArrowLeft} alt="arrow_left" />
+                                        ) : (
+                                            <img src={ArrowRight} alt="arrow_right" />
+                                        )}
+                                    </ImageContainer>
+                                    <ContentWrapper>
+                                        {item.speakers.map((item) => {
+                                            return (
+                                                <>
+                                                    <Header
+                                                        as="div"
+                                                        align="left"
+                                                        type="subtitle-1"
+                                                        weight="bold"
```

Comment:
We don't need this due the default properties

Reply:
okay